### PR TITLE
feat: lw13 tiger d5

### PR DESCRIPTION
**Pull request by w3b6x9**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

database.build v2

### CODE REVIEW

File: apps/www/_blog/2024-12-06-database-build-v2.mdx [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+  />
+</video>
+
+This can be useful if you have existing schema or migrations that were designed elsewhere that you wish to extend or ask questions about.
+
+## Mobile support
+
+We’re excited to share that database.build now works on mobile! Whether you’re on the train, stuck in a meeting, or lounging on the couch, you can spin up databases and tweak your schema with just a few taps.
+
+![database.build v2 mobile support](/images/blog/launch-week-13/day-5-database-build-v2/mobile-support.png)
```
Comment (w3b6x9) [inline apps/www/_blog/2024-12-06-database-build-v2.mdx:208]: @gregnr i used this image from the `Redesign` section here

File: apps/www/_blog/2024-12-06-database-build-v2.mdx [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+
+With mobile support, you officially have no excuse not to ship that feature you’ve been procrastinating on. We’re excited to see what you build!
+
```
Comment (w3b6x9) [inline apps/www/_blog/2024-12-06-database-build-v2.mdx:211]: @gregnr i cut out the `Redesign` section b/c it felt redundant given that the other vids and screenshots already contained the new redesign and i used the mobile image above in the `Mobile support` section.

feel free to add it in though.

File: apps/www/_blog/2024-12-06-database-build-v2.mdx [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+  - postgres
+tags:
+  - launch-week
+  - AI
+  - postgres
+date: '2024-12-06T00:00:03'
+toc_depth: 3
+launchweek: '13'
+---
+
```
Comment (w3b6x9) [inline apps/www/_blog/2024-12-06-database-build-v2.mdx:19]: @gregnr REMINDER: drop your launch video somewhere in here when ready.

File: apps/www/_blog/2024-12-06-database-build-v2.mdx [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+Finally you will be presented with connection strings and a password to connect to your new database.
+
+### Behind the scenes
+
+Getting deployments to work took quite a bit more engineering than it might seem at first glance. To copy your in-browser database to the cloud accurately, we needed a way to dump and restore. ElectricSQL has been hard at work on a WASM version of `pg_dump` that can run directly in the browser—a mind-blowing feat in its own right—but unfortunately, it wasn’t quite ready in time for this launch.
+
+In the meantime, we decided to piggyback on top of our existing Live Share feature, which creates a reverse connection between your in-browser database and an external PostgreSQL client. With Live Share enabled, we connect a server-side `pg_dump` program directly to your in-browser database. The resulting dump is then piped into `pg_restore`, which transfers it to your new cloud database.
+
+![database.build v2 Deploy to Supabase](/images/blog/launch-week-13/day-5-database-build-v2/deploy-to-supabase-dark.png)
```
Comment (w3b6x9) [inline apps/www/_blog/2024-12-06-database-build-v2.mdx:174]: @fsansalvadore i couldn't get this diagram right. need to make sure it looks good in both dark and light modes.

File: apps/www/_blog/2024-12-06-database-build-v2.mdx [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+
+[Service workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API) to the rescue! One of the core features of service workers is the ability to intercept fetch requests and handle them client side - exactly what we need. With this approach, we can simply leave our `useChat()` logic as-is, then create a lightweight service worker that does the following:
+
+1. Detects whether you’re using your own model by reading local state from [IndexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API) (service workers don’t have access to local storage, so we opt for IndexedDB).
+1. If detected, it serves a local HTTP handler on `/api/chat`, sending the request directly to your provider’s API.
+1. Otherwise, it forwards the request to our backend as it did before.
+
+![database.build v2 BYO-LLM API key](/images/blog/launch-week-13/day-5-database-build-v2/llm-service-worker-dark.png)
```
Comment (w3b6x9) [inline apps/www/_blog/2024-12-06-database-build-v2.mdx:104]: @fsansalvadore this too. need to make sure it looks good in both dark and light modes.

**Review by fsansalvadore — APPROVED** [review]

File: apps/www/_blog/2024-12-06-launch-week-13-top-10.mdx [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+Want to test hacking skills? We're running a Capture the Flag event called [Hack the Base](https://ctf.supabase.com/). It's a free event that anyone can participate in.
+
+[Read more →](/blog/launch-week-13-hackathon)
+
+### Launch Week 13: Hackathon
+
+We're running a virtual hackathon starting **now** where you can win prizes for building the best projects using Supabase. We'll be announcing the winners on December 15th.
+
+[Read more →](/blog/launch-week-13-hackathon)
```
Comment (w3b6x9) [inline apps/www/_blog/2024-12-06-launch-week-13-top-10.mdx]: @kiwicopple

```suggestion
[Read more →](/events/launch-week-13-hackathon)
```

**Pull request merged**